docs/learn/networks.md

```diff
@@ -23,9 +23,25 @@ For more information about Swanky Node, check out the [Swanky Suite](https://doc
 
 With Zombienet users can download arbitrary Relay Chain and parachain binaries (or use images) to setup a configurable local test network. Users will have access to all privileged actions on the Relay Chain and parachains, which simplifies testing. For more information about Zombienet, check out the [Build Environment](https://docs.astar.network/docs/build/environment/zombienet-testing) chapter.
 
+### zkEVM Node
+
+A local Validium instance based on Polygon CDK that includes the following components:
+
+- zkEVM databases: data node, event, explorer L1 and L2, pool, state, and bridge service
+- zkEVM node components: aggregator, approve service, sequencer and sequence sender, synchronizer
+- L1 network (mock)
+- Prover
+- Explorers L1, L2
+- JSON RPC explorer
+- L2 gas pricer
+- DAC: data availability service, dac setup committee
+- zkEVM bridge service and UI
```

Comment:
A lot of wrong info. Please refer to https://github.com/0xPolygon/cdk-validium-node?tab=readme-ov-file#key-components and re-write

Reply:
this is taken directly from https://docs.astar.network/docs/build/zkEVM/zk-node/setup-local-validium-node so one of our devs will need to provide the correct information for both sections.

Reply:
for now, this section has been removed.